### DISCUSSION TITLE
Re-Building the Set Analysis Wizard

### DISCUSSION THREAD

**Discussion by stefanwalther** [stefanwalther/set-analysis-wizard#6] [Announcements]
Hi folks,

it's now many, many years ago - actually back in the year 2009 - that I build the "Set Analysis Wizard" to help people understand the Set Analysis syntax. Since then a lot of things have changed; last but not least I am not working in the Qlik eco-system anymore. But I still tried to keep all tooling and other Qlik-related resources alive ... I tried ...

Unfortunately, a few weeks ago the Set Analysis Wizard discontinued working. Yes, it's all based on pretty old 2005-ish technology like Asp.net, .aspx pages, and even some C# was involved.

After trying for a few weeks to find a new hoster, I gave up and decided to

REBUILD the entire SET ANALYSIS WIZARD ...

You might ask why I am doing this. The answer is simple, it's just my tribute to the Qlik community I loved to work with for over 15 years. Furthermore, I am amazed at how many people still try to use the Set Analysis Wizard every day, so I was even more surprised that I got more than 200 emails & messages in the last six weeks asking about the wizard ... 😊 😁 😳

So just stay tuned; in a few weeks, you'll find the new Set Analysis Wizard here on this repo (new Url will then be https://stefanwalther.github.io/set-analysis-wizard), but I will also redirect the old Url (...tools.qlikblog.at ...) ...

All the best, Stefan

P.S.: While you are waiting: I have created a few short polls related to the new version of the Set Analysis Wizard. It would be great if you could spend some minutes answering [them here](https://github.com/stefanwalther/set-analysis-wizard/discussions/categories/polls).

P.P.S: You can follow progress [here](https://github.com/users/stefanwalther/projects/16/views/1) ...

**Comment by countnazgul** [comment]
Thanks for the support Stefan!

Just to mention that Qlik have just released new set analysis syntax 

https://community.qlik.com/t5/Qlik-Design-Blog/New-Set-Analysis-syntax/ba-p/1983110

**Reply by stefanwalther** [reply]
@countnazgul Yep, seen it, thx. Not sure how I will get those additions reflected, but good news - in any case - is that the "old" syntax will continue to work.

**Comment by muellermarco** [comment]
Hello Stefan,

Your wizard made my initial hurdle to Set-Analysis much easier and even now with experience I am always happy when I can generate more complex expressions shortly. For new Qlik users, I always recommend the link to your site to practice their Set skills. 

So I am extremely grateful that you continue to support the community even though you are no longer directly involved. Thanks a lot!

Let me know if I can be of any assistance.

Cheers,
Marco

**Reply by stefanwalther** [reply]
Thank you for your kind words.

Yes, you can definitely help. The first version of the re-build (functionally fully working, but not pretty) should be online soon. Any help to test it would be greatly appreciated!

**Comment by m3cowley** [comment]
So happy to hear, Stefan.

I've both used this myself, and directed anyone new to Qlik to this resource.  I've often wondered why Qlik didn't buy it from you!

Very much appreciated,
Matt

**Reply by stefanwalther** [reply]
It wasn't even about buying; I offered it for free ... but the value was just not seen ... although I can see in the stats of the last few years how many people used it every day ...

**Reply by GMuck** [reply]
Hi Stefan, Is there a final version for the Set Analysis Wizard?  Thank you! Greg

**Comment by daviewales** [comment]
The link to the new wizard is broken, but you can still access the old one here:
https://web.archive.org/web/20220519141800/http://tools.qlikblog.at/SetAnalysisWizard/QlikView-SetAnalysis_Wizard_and_Generator.aspx?sa=
